src/Entities/Template/EntityDetails.jsx

```diff
@@ -186,6 +186,11 @@ const entityProperties = (obj) => {
 const listLink = href({
   widgetSrc: returnTo,
 });
+const detailsLink = href({
+  widgetSrc: `${REPL_ACCOUNT}/widget/Entities.Template.EntityDetails`,
+  params: { src, schemaFile, namespace, entityType, returnTo: homeLink },
+});
+const detailsUrl = "https://${REPL_NEAR_URL}/" + detailsLink;
```

Comment:
```suggestion
const detailsUrl = `https://${REPL_NEAR_URL}/${detailsLink}`;
```

This suggestion will work the same but looks cleaner

Reply:
I used concatenation here to highlight that the replacement of REPL_NEAR_URL is not via ES6 template. Also, it should be slightly faster, although we don't really care about performance here.